lib/changelogerator.rb

```diff
@@ -1,5 +1,49 @@
 # frozen_string_literal: true
 
+# Return the label code for a change
+# if the label name matches the expected pattern.
+# nil otherwise.
+def parse_change_label(name)
+  m = match = name.match(/^([a-z])(\d+)-(.*)$/i)
+  return nil unless m
+
+  letter, digits, text = match.captures
+  number = digits.to_i
+  [letter, number, text]
+end
+
+def compute_change_meta(change)
```

Comment:
doc ?

Reply:
It was removed by accident. Re-added from https://github.com/paritytech/changelogerator/blob/81e8a9639704de1b13cb3d488647085614b8368c/lib/changelogerator.rb#L128.